### PR TITLE
Discover page

### PR DESCRIPTION
**Pull request by basticks02**
In this PR, I will be pulling covid-19 resources from the Open-Diseases Data API and rendering them on the Discover page.

This is the layout for the page: 
Screenshot 2024-07-09 at 3 02 56 PM

### CODE REVIEW

**Comment by basticks02** [comment]
https://www.loom.com/share/51d066f8ad764c9d8876910afc8a2512?sid=30b29ebf-7110-4ff1-9aac-b98240a628c4

**Comment by basticks02** [comment]
https://www.loom.com/share/3b168445ef124b9b883524911f8c1035?sid=b7b0fbad-21d8-4111-97c5-d4511a4705e8

File: frontend/src/Discover/Map.jsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      const response = await fetch('/countries.geojson');
+      return response.json();
+    };
+
+    const fetchData = async () => {
+      try {
+        const response = await axios.get('https://disease.sh/v3/covid-19/vaccine/coverage/countries?lastdays=1');
+        const data = response.data;
+
+        const vaccineData = {};
```
Comment (cherieliu) [inline frontend/src/Discover/Map.jsx:46]: could we leave some more high-level comments on what the different pieces of logic are? to make the code a bit more understandable

**Review by cherieliu — COMMENTED** [review]
overall looks rlly great!!

**Pull request merged**